webapp/channels/src/components/admin_console/team_channel_settings/convert_confirm_modal.tsx

```diff
@@ -115,10 +128,11 @@ const messages = defineMessages({
     },
     toPublicMessage: {
         id: 'admin.team_channel_settings.convertConfirmModal.toPublicMessage',
-
-        // This eslint-disable comment can be removed once this component no longer uses FormattedMarkdownMessage
-        // eslint-disable-next-line formatjs/no-multiple-whitespaces
-        defaultMessage: 'When you convert **{displayName}** to a public channel, history and membership are preserved. Public channels are discoverable and can by joined by users on the system without invitation.  \n \nAre you sure you want to convert **{displayName}** to a public channel?',
+        defaultMessage: 'When you convert {displayName} to a public channel, history and membership are preserved. Public channels are discoverable and can by joined by users on the system without invitation.',
```

Comment:
There is a typo (from before this PR). Do you mind correcting it?

```suggestion
        defaultMessage: 'When you convert {displayName} to a public channel, history and membership are preserved. Public channels are discoverable and can be joined by users on the system without invitation.',
```